lerobot/common/datasets/utils.py

```diff
@@ -126,6 +129,21 @@ def load_episode_data_index(repo_id, version, root) -> dict[str, torch.Tensor]:
     return load_file(path)
 
 
+def get_hf_dataset_safe_version(repo_id: str, version: str) -> str:
+    dataset_info = list_repo_refs(repo_id, repo_type="dataset")
+    branches = [b.name for b in dataset_info.branches]
+    if version not in branches:
+        warnings.warn(
+            f"Version '{version}' not found on {repo_id}. Using 'main'.",
```

Comment:
Ideally:
```suggestion
            f"You are trying to load a dataset from {repo_id} created with a previous version of the codebase ('{dataset_version}'). The requested version ('{requested_version}') is not found. You should be fine since backward compatibility is maintained. If you encounter a problem, contact LeRobot maintainers on Discord ('https://discord.com/invite/s3KuuzsPFb') or open an issue on github.",
```

Reply:
Added https://github.com/huggingface/lerobot/pull/318/commits/3831f4ce22e6de422c0f1b663a4f797450796a78